src/App.vue

```diff
@@ -161,7 +161,7 @@ import * as Dialogs from '@/dialogs/'
 import { AuthServices } from '@/services/'
 import { CommonMixin, DateMixin, FilingTemplateMixin, LegalApiMixin } from '@/mixins/'
 import { FilingDataIF, ActionBindingIF, ConfirmDialogType, FlagsReviewCertifyIF, FlagsCompanyInfoIF,
-  AlterationFilingIF, FirmChangeIF, FirmConversionIF } from '@/interfaces/'
+  AlterationFilingIF, ChangeFilingIF, ConversionFilingIF } from '@/interfaces/'
```

Comment:
This PR includes commits that are already in main branch. Was there an issue while rebasing?

Reply:
Yes there was, should I roll it back a commit to fix this?

Reply:
Its strange I found some other people have had this issue and I found a workaround that involved changing a the base branch to something other than main and then change the branch back to main, then GitHub registered the change. It seems to have worked.